dask/bytes/tests/test_http.py

```diff
@@ -12,6 +12,10 @@
 
 files = ["a", "b"]
 requests = pytest.importorskip("requests")
+errs = (requests.exceptions.RequestException,)
+if LooseVersion(fsspec.__version__) > "0.7.4":
+    aiohttp = pytest.importorskip("aiohttp")
```

Comment:
If we do this and aiohttp is not available won't this skip all the tests?

Reply:
Yes, but it really is required for HTTP on fsspec for the release coming soon

Reply:
Ah, but maybe you mean that aiohttp should be in the test env .. a good point.